v2/dtos/common/ping.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// PingRequest defines the request associated with the /ping endpoint.
+// This object and its properties correspond to the PingRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/PingRequest
+type PingRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
+}
+
+// PingResponse defines the content of response content for POST Ping DTO
+// This object and its properties correspond to the Ping object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/PingResponse
+type PingResponse struct {
+	BaseResponse `json:",inline"`
+	Timestamp    string `json:"timestamp"`
```

Comment:
From recent testing it would be nice if ping returned the ServiceKey. I know this is a change to the API spec, but I think it is more useful than `Timestamp`. @tonyespy, @jpwhitemn  thoughts?